_static/author_bio.css

```
@@ -0,0 +1,30 @@
+.bio {
+    display: flex;
+    margin: 1em 0;
+}
+.photo {
+    max-width: 60px;
+    margin-right: 1em;
+}
+.photo__img {
+    width: 100%;
+    border-radius: 50%;
+}
+.bio-text__h4 {
```

```suggestion
.bio-text__author-name {
```
I think this class name could be more general and less specific to the element it is being added to, because if we wanted to change the element from `h4` to `h3`, we would need to change the class name as well.

---

Done!